File: src/JVM/label/codeLabel.cpp
#include <iostream>
#include "codeLabel.h"

namespace toyc {

  codeLabel::codeLabel(std::string n) { name = n; }
  std::string codeLabel::getLabel() { return name; }
  std::string codeLabel::toString() { return name + ":"; }
}

File: src/JVM/label/codeLabel.java
package codegen.JVM.label;

import compilers.LineOfCode;
import codegen.JVM.JVMlabel;

public class codeLabel implements JVMlabel, LineOfCode {
    private String name;

    public codeLabel(String name) { this.name = name; }
  
    public String toString() { return name + ":"; }
}
